Classes/BEMSimpleLineGraphView.h

```diff
@@ -449,13 +449,13 @@ IB_DESIGNABLE @interface BEMSimpleLineGraphView : UIView <UIGestureRecognizerDel
 /** The optional suffix to append to the popup report.
  @param graph The graph object requesting the total number of points.
  @return The suffix to append to the popup report. */
-- (NSString *)popUpSuffixForlineGraph:(BEMSimpleLineGraphView *)graph;
+- (NSString *)popUpSuffixForlineGraph:(BEMSimpleLineGraphView *)graph index:(NSUInteger)index;
 
 
 /** The optional prefix to append to the popup report.
  @param graph The graph object requesting the total number of points.
  @return The prefix to prepend to the popup report. */
-- (NSString *)popUpPrefixForlineGraph:(BEMSimpleLineGraphView *)graph;
+- (NSString *)popUpPrefixForlineGraph:(BEMSimpleLineGraphView *)graph index:(NSUInteger)index;
```

Comment:
Please rename the method to  `- (NSString *)popUpPrefixForlineGraph:(BEMSimpleLineGraphView *)graph atIndex:(NSUInteger)index;`.